src/main/java/com/hillel/items_exchange/service/AdvertisementService.java

```diff
@@ -116,6 +118,20 @@ public void remove(long id) {
         advertisementRepository.flush();
     }
 
+    public void setDefaultImage(Long advertisementId, Long imageId, User owner) throws ClassNotFoundException {
+        final Advertisement advertisement = owner.getAdvertisements().stream()
+                .filter(adv -> adv.getId() == advertisementId)
+                .findFirst().orElseThrow(() -> new ClassNotFoundException(
+                        getExceptionMessageSource("exception.illegal.id")));
+        final Image image = advertisement.getProduct().getImages().stream()
+                .filter(img -> img.getId() == imageId)
+                .findFirst().orElseThrow(() -> new ClassNotFoundException(
```

Comment:
Move .orElseThrow() to new line and that will allow you to create the exception on that line. (here and upper)